### ISSUE TITLE
Verify SigmaE / E Systematic Impact 

### ISSUE THREAD

**Issue by atishelmanch** [atishelmanch/flashgg#8]
Should the up/down variation of the SigmaE / E systematic have an affect on the CMS_hgg_mass? Currently we see it does not. Need to understand the expected impact and make sure what we see makes sense.

Asignee: @chuwang1

**Comment by atishelmanch** [comment]
To be clear, looks there there are both the "Smearing" and "Shift" systematics for SigmaE/E, where Shift has up/down variations and Smearing does not by default in our current version of the repo. 

-------------- DUMPING SYSTEMATIC OVERVIEW --------------
          Systematic  Central value?   Systematic shifts?
---------------------------------------------------------
 SigmaEOverESmearing             YES                   NO
    SigmaEOverEShift              NO                -1 1

**Comment by chuwang1** [comment]
Yes, I have noticed this. SigmaEoverE just shift to up and down, but not apply to central value.
https://github.com/atishelmanch/flashgg/blob/master/Systematics/plugins/PhotonSigEoverEShift.cc#L48-L50
